recipes/djangorestframework-filters/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "djangorestframework-filters" %}
+{% set version = "1.0.0.dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d53692f9f0dfcdc1df4e890787f7212c3602476b85faffdefec02e2bc386c5b6
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+  run:
+    - django-filter >=2.0
+    - djangorestframework
+    - python >=3.4
+
+test:
+  imports:
+    - rest_framework_filters
+    - rest_framework_filters.templatetags
+
+about:
+  home: http://github.com/philipn/django-rest-framework-filters
+  license: BSD-3-Clause
```

I think this license here is ``MIT``

You still need to package it, I ran [grayskull](https://github.com/marcelotrevisani/grayskull/) to generate this recipe and it generated as MIT
I checked their license on pypi and it is pointing as MIT as well. On their ``README`` there is a section there called license and they just copied the MIT license there
```suggestion
  license: MIT
```

---

Please see
https://github.com/philipn/django-rest-framework-filters#license